EpikCord/__init__.py

```diff
@@ -2305,6 +2305,65 @@ def latency(self):
     def average_latency(self):
         return sum(self.latencies) / len(self.latencies)
 
+    def add_task(self, task: Callable, interval: Optional[int] = None, **kwargs):
+        """Adds a background task (Tasks that run silently in the background)
+
+        This task can be used to monitor the bot's performance, change and send stuff routinely
+        through webhooks, etc
+
+        You can set the time of when it starts, stops and  add which arguments to give to the task
+
+        You may also define whether to run once or run indefinitely
+
+        Args:
+            task (Callable): The function to run on the background
+            interval (int, optional): The interval (in seconds) for a delay between when the instance of the task stops and starts again .Defaults to 5.
```

Comment:
The typehint should be like `interval: Optional[int]`, not that

Reply:
done